File: Examples/CSharp/AsposePDF/Working-Document/GetXMPMetadata.cs
//////////////////////////////////////////////////////////////////////////
// Copyright 2001-2013 Aspose Pty Ltd. All Rights Reserved.
//
// This file is part of Aspose.Pdf. The source code in this file
// is only intended as a supplement to the documentation, and is provided
// "as is", without warranty of any kind, either expressed or implied.
//////////////////////////////////////////////////////////////////////////
using System.IO;

using Aspose.Pdf;
using System;

namespace CSharp.AsposePdf.WorkingDocuments
{
    public class GetXMPMetadata
    {
        public static void Run()
        {
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_AsposePdf_WorkingDocuments();

            //open document
            Document pdfDocument = new Document(dataDir + "GetXMPMetadata.pdf");

            //get properties
            Console.WriteLine(pdfDocument.Metadata["xmp:CreateDate"]);
            Console.WriteLine(pdfDocument.Metadata["xmp:Nickname"]);
            Console.WriteLine(pdfDocument.Metadata["xmp:CustomProperty"]);

        }
    }
}

File: Examples/VisualBasic/AsposePDF/Working-Document/GetXMPMetadata.vb
Imports System.IO
Imports System
Imports Aspose.Pdf

Namespace AsposePDF.WorkingDocuments
    Public Class GetXMPMetadata
        Public Shared Sub Run()
            ' ExStart:GetXMPMetadata
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_AsposePdf_WorkingDocuments()

            ' Open document
            Dim pdfDocument As New Document(dataDir & Convert.ToString("GetXMPMetadata.pdf"))

            ' Get properties
            Console.WriteLine(pdfDocument.Metadata("xmp:CreateDate"))
            Console.WriteLine(pdfDocument.Metadata("xmp:Nickname"))
            Console.WriteLine(pdfDocument.Metadata("xmp:CustomProperty"))
            ' ExEnd:GetXMPMetadata
        End Sub
    End Class
End Namespace
